apertium-tur-uzb.tur-uzb.dix

```diff
@@ -238,6 +238,7 @@
     <e><p><l>ve<s n="cnjcoo"/></l><r>va<s n="cnjcoo"/></r></p></e>
     <e><p><l>ise<s n="cnjcoo"/></l><r>esa<s n="cnjcoo"/></r></p></e>
     <e><p><l>da<s n="cnjcoo"/></l><r>ham<s n="cnjcoo"/></r></p></e>
+    <e><p><l>de<s n="cnjcoo"/></l><r>ham<s n="cnjcoo"/></r></p></e>
```

Comment:
This shouldn't be needed.  Any time "de" is encountered, it should be analysed as `da<cnjcoo>`.  If it doesn't, that's a problem with tur.lexc.

Reply:
Deleted it then.